src/components/Colors/Color.tsx

```diff
@@ -7,41 +7,152 @@ import { ComponentPropsWithoutRef } from "react";
 export interface ColorRectangleProps extends ComponentPropsWithoutRef<"div"> {
   theme: string;
 }
+const baseClass = "w-40 h-10 mb-2";
+function Color({ color }: { color: string }) {
+  return (
+    <div
+      id={color}
+      className={`${baseClass} ${color} border-2 border-black-600`}
```

Comment:
`border-2 border-black-600`

Not following here... what's that!?
🤔

Reply:
@Pierregvx @mazmassa I think we can use this
```suggestion
      className={`${baseClass} ${color} border-2 border-c-default`}
```

Reply:
> @Pierregvx @mazmassa I think we can use this

But test it and tell me if it works well with black and white themes.

Reply:
The question remains: why do we need to set up a border color!?

Can we have a screen of what is!?

Reply:
![image](https://github.com/massalabs/ui-kit/assets/72019005/e457a853-a460-4e67-8b82-022a33979614)

For this

Reply:
Got it! Thanks for the explanation @maxime-carabina 

You are right, we should use c-default!